docs/tutorials/get-started/blink-an-led.md

```diff
@@ -124,27 +127,25 @@ Now that your circuit is wired, reconnect your Pi to power.
 Before proceeding, be sure that you have [connected your Pi to the Viam app](/get-started/installation/#install-viam-server).
```

Comment:
```suggestion
Before proceeding, be sure that you have [connected your Pi to the Viam app](#project-setup).
```
[opt] So that they know it's instructions they've already completed if they've been following along